docs/USAGE-GUIDE.md

```diff
@@ -703,8 +715,6 @@ or low latency. Connections prefering low latency will be encrypted using small
 record sizes that can be decrypted sooner by the recipient. Connections
 prefering throughput will use large record sizes that minimize overhead.
 
-Connections prefer low latency by default.
```

Comment:
Good idea. thanks

`**s2n_connection_prefer_throughput** and **s2n_connection_prefer_low_latency**
change the behavior of s2n when sending data to prefer either throughput
or low latency. Connections prefering low latency will be encrypted using small
record sizes that can be decrypted sooner by the recipient. Connections
prefering throughput will use large record sizes that minimize overhead.

-Connections are defaulted to 8k bytes.`